docs/vocascan-server/contribute.md

```diff
@@ -36,15 +38,16 @@ Copy the env config file template to configure your server.
 cp vocascan.config.example.js vocascan.config.js
 ```
 
-Fill out the config file. For a complete guide for each option see [configuration](vocascan-server/configuration)
+Fill out the config file.`server.jwt_secret` and `database.*` needs to be changed. For a complete guide for each option
```

Comment:
missing space (check also the other files where you used that phrase.

Reply:
Can't see a missing space here